### PR TITLE
fix: update apis-manager doc

### PR DESCRIPTION
**Pull request by mrednic-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The documentation is sorted by topic but in the table below, you shall find the
...
+| @o3r/apis-manager        | Service to help you communicate with your APIs                                                                                                                 | @o3r/dev-tools                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (matthieu-crouzet) [inline docs/README.md:11]:
```suggestion
| @o3r/apis-manager        | Service to help you communicate with your APIs                                                                                                                 | @o3r/dev-tools                                                                                                                                                                                               | [API_MANAGER](https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/apis-manager/README.md)                                                                                                                                                                                                                                                                                                                                                     |
```

Comment (matthieu-crouzet) [reply docs/README.md:11]: Usually we write this one I think

Comment (mrednic-1A) [reply docs/README.md]: updated :)

File: packages/@o3r/apis-manager/README.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ This package is an [Otter Framework Module](https://github.com/AmadeusITGroup/ot
 [![Stable Version](https://img.shields.io/npm/v/@o3r/apis-manager?style=for-the-badge)](https://www.npmjs.com/package/@o3r/apis-manager)
 [![Bundle Size](https://img.shields.io/bundlephobia/min/@o3r/apis-manager?color=green&style=for-the-badge)](https://www.npmjs.com/package/@o3r/apis-manager)
 
-This module provides services to help you communicate with your APIs. Its responsibility is to provide an API configuration to a service factory so that it could instantiate an API with the right configurations.
+This module is doing the link between the SDK generated based on your APIs using the [Otter generator](https://www.npmjs.com/package/@ama-sdk/schematics), and your application.
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:15]:
```suggestion
This module links the SDK generated from your APIs using the [Otter generator](https://www.npmjs.com/package/@ama-sdk/schematics) to your application.
```

File: packages/@o3r/apis-manager/README.md [hunk 3]
```diff
@@ -12,7 +12,9 @@ This package is an [Otter Framework Module](https://github.com/AmadeusITGroup/ot
...
+In simpler terms, it offers services to facilitate communication with your APIs.
+Its responsibility is to provide an API configuration to a service factory. This enables the factory to create an API instance with the right configurations.
 
 It contains a default configuration and a map of specific configurations for API / set of API.
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:19]:
```suggestion
It contains a default configuration and a map of specific configurations for an API or a set of APIs.
```

File: packages/@o3r/apis-manager/README.md [hunk 4]
```diff
@@ -29,13 +31,19 @@ ng add @o3r/apis-manager
 
 ## Usage
 
-The API Manager Module will need to be imported and configured in the **application module** which will then be used by the **ApiFactory**.
+The API Manager Module will need to be imported and configured at **application level**, which will then be used by the **ApiFactory** service.
 
 ### Application side configuration
 
-The **ApiManager** requires the default Api Client which will be used in all the APIs and support a second parameter that allows the user to define specific Api Client per APIs.
+The **ApiManager** requires the default Api Client, which will be utilized across all APIs. It supports a second parameter that allows to define specific Api Client per APIs.
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:38]:
```suggestion
The **ApiManager** requires the default API Client, which will be used across all APIs. It supports a second parameter that allows to define specific API Client configurations to set or override per API.
```

File: packages/@o3r/apis-manager/README.md [hunk 5]
```diff
@@ -29,13 +31,19 @@ ng add @o3r/apis-manager
...
 
+In the example that follows, we define the default base configuration that API classes will use, as well as a custom configuration for the 'ExampleApi'.
+The plugins and fetch client are from _@ama-sdk/core_ module, but custom ones can be created if needed. More details on _@ama-sdk/core_ [here](https://www.npmjs.com/package/@ama-sdk/core).
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:41]:
```suggestion
The plugins and fetch client are from the _@ama-sdk/core_ module, but custom ones can be created if needed. More details on _@ama-sdk/core_ [here](https://www.npmjs.com/package/@ama-sdk/core).
```

File: packages/@o3r/apis-manager/README.md [hunk 6]
```diff
@@ -112,13 +127,10 @@ class MyClass {
 }
 ```
 
-> [!WARNING]
-> Do not forget to import the **ApiManagerModule** in you component module
-
 ### Enforce custom API usage
 
 Some users may want to enforce existing components or services to use a specific Sdk instead of default API SDK.
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:132]:
```suggestion
Some users may want to enforce existing components or services to use a specific SDK instead of the default API SDK.
```

File: packages/@o3r/apis-manager/README.md [hunk 7]
```diff
@@ -112,13 +127,10 @@ class MyClass {
...
-To do so the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory** the class they will need to use (instead of default ones).
+To do so, the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory**, the class they will need to use (instead of default ones).
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:133]:
```suggestion
To do so, the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory** which class it must use (instead of default ones).
```

File: packages/@o3r/apis-manager/README.md [hunk 8]
```diff
@@ -112,13 +127,10 @@ class MyClass {
...
 
 In the AppModule:
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:136]: lines 142-144: providers is an array so replace the {} by []

**Comment by mrednic-1A** [comment]
@sdo-1A @matthieu-crouzet Thanks a lot for having a look.
FYI I have fixed all your comments.

File: packages/@o3r/apis-manager/README.md [hunk 9]
```diff
@@ -29,21 +31,27 @@ ng add @o3r/apis-manager
 
 ## Usage
 
-The API Manager Module will need to be imported and configured in the **application module** which will then be used by the **ApiFactory**.
+The API Manager Module (`ApiManagerModule`) needs to be imported and configured at **application level**, which will then be used by the **ApiFactory** service.
 
 ### Application side configuration
 
-The **ApiManager** requires the default Api Client which will be used in all the APIs and support a second parameter that allows the user to define specific Api Client per APIs.
+The **ApiManager** requires the default API Client, which will be used across all APIs. It supports a second parameter that allows to define specific API Client to set or override per API.
```
Comment (sdo-1A) [inline packages/@o3r/apis-manager/README.md:38]:
```suggestion
The **ApiManager** requires the default API Client, which will be used across all APIs. It supports a second parameter that allows to define specific a API Client to set or override per API.
```

OR 

```suggestion
The **ApiManager** requires the default API Client, which will be used across all APIs. It supports a second parameter that allows to define specific API Client configurations to set or override per API.
```

File: packages/@o3r/apis-manager/README.md [hunk 10]
```diff
@@ -29,21 +31,27 @@ ng add @o3r/apis-manager
...
 
+In the example that follows, we define the default base configuration that API classes will use, as well as a custom configuration for the 'ExampleApi'.
+The plugins and fetch client are from the _@ama-sdk/core_ module, but custom ones can be created if needed. More details on _@ama-sdk/core_ [here](https://www.npmjs.com/package/@ama-sdk/core).
```
Comment (cpaulve-1A) [inline packages/@o3r/apis-manager/README.md:41]:
```suggestion
The plugins and fetch client come from the ``@ama-sdk/core`` module, but custom ones can be created if needed as long as they follow the ``ApiClient`` interface from ``@ama-sdk/core``. More details on ``@ama-sdk/core`` [here](https://www.npmjs.com/package/@ama-sdk/core).
```

File: packages/@o3r/apis-manager/README.md [hunk 11]
```diff
@@ -112,13 +127,10 @@ class MyClass {
...
 
-> [!WARNING]
-> Do not forget to import the **ApiManagerModule** in you component module
-
 ### Enforce custom API usage
 
-Some users may want to enforce existing components or services to use a specific Sdk instead of default API SDK.
-To do so the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory** the class they will need to use (instead of default ones).
+Some users may want to enforce existing components or services to use a specific SDK instead of the default API SDK.
+To do so, the **INITIAL_APIS_TOKEN** will allow to indicate to the **ApiFactory** which class it must use (instead of default ones).
```
Comment (cpaulve-1A) [inline packages/@o3r/apis-manager/README.md:133]:
```suggestion
To do so, the ``INITIAL_APIS_TOKEN`` will allow to indicate to the ``ApiFactory`` which class it must use instead of default ones.
```

File: packages/@o3r/apis-manager/README.md [hunk 12]
```diff
@@ -29,21 +31,27 @@ ng add @o3r/apis-manager
...
+const PROXY_SERVER = "https://your-enpoint-base-path";
 export const apiManager = new ApiManager(
   new ApiFetchClient({
     basePath: PROXY_SERVER,
     requestPlugins: [new ApiKeyRequest('YourApiKey', 'Authorization')]
   }),
 
   {
-    ExampleApi: // <--custom config for ExampleApi, using jsonToken plugins. If fields are not provided, the default ones will be taken.
+    ExampleApi: // <-- custom config for ExampleApi, using jsonToken plugins. If fields are not provided, the default ones will be used.
```
[the remark this answers is not in the document]
Comment (mrednic-1A) [reply packages/@o3r/apis-manager/README.md:54]: The one define in line 48 applies for all APIs, so that's the reference here. I will update to specify this clear. :)

**Review by matthieu-crouzet — APPROVED** [review]

File: packages/@o3r/apis-manager/README.md (hunk 9 above)
Comment (mrednic-1A) [reply packages/@o3r/apis-manager/README.md]: updated with the first proposal

**Review by kpanot — APPROVED** [review]
